config/default.json

```diff
@@ -16,7 +16,7 @@
     "availableCacheTags": ["product", "category", "home", "checkout", "page-not-found", "compare", "my-account", "P", "C", "error", "attribute", "taxrule"],
     "invalidateCacheKey": "aeSu7aip",
     "invalidateCacheForwarding": false,
-    "invalidateCacheForwardUrl": "http://localhost:8080/invalidate?key=aeSu7aip&tag=",
+    "invalidateCacheForwardUrl": "https://next.storefrontcloud.io/invalidate?key=aeSu7aip&tag=",
```

Comment:
endpoints should be restored to localhost in default

Reply:
yes, it was only needed for netlify. I reverted it